### PR TITLE
Accept `next/font/google` as well as `@next/font/google`

### PR DESCRIPTION
**Pull request by wbinnssmith**
Aligning with https://github.com/vercel/next.js/pull/45707, this allows users to import fonts from `next/font/google` as well as `@next/font/google`.

Test Plan: Adjusted tests to use `next/font/google` and added another test to verify `@next/font/google` continues to work.

### CODE REVIEW

File: crates/next-core/src/next_font_google/mod.rs [hunk 1]
```diff
@@ -82,14 +82,19 @@ impl ImportMappingReplacement for NextFontGoogleReplacer {
...
                                 style: {{
                                     fontFamily: "{}",
                                     {}{}
                                 }},
-                                variable: cssModule.variable
                             }};
+
+                            if (cssModule.variable != null) {{
+                                fontData.variable = cssModule.variable;
+                            }}
```
Comment (jridgewell) [inline crates/next-core/src/next_font_google/mod.rs:95]: Why is this change included?

**Review by jridgewell — APPROVED** [review]

**Review by hanneslund — APPROVED** [review]

File: crates/next-core/src/next_font_google/mod.rs (hunk 1 above)
Comment (wbinnssmith) [reply crates/next-core/src/next_font_google/mod.rs:95]: It's unrelated, but a small issue issue I fixed while adding the new test. Could have been a new PR I suppose.
